geps/gep-1709.md

```diff
@@ -37,8 +45,304 @@ which implementations can prove they satisfy and be recognized for.
+features are implemented (as described in the [profiles](#profiles) section
+above):
+
+```console
+$ go test ./conformance/... -args \
+    -gateway-class=acme \
+    -conformance-profiles=Layer7,Layer4 \
+    -unsupported-features=HTTPResponseHeaderModification,HTTPRouteMethodMatching,HTTPRouteQueryParamMatching,
+```
+
+Or the equivalent configuration using the Golang library:
+
+```go
+cSuite, err := suite.New(suite.Options{
+    GatewayClassName: "acme",
+    Profiles: sets.New(
+        Layer7,
+        Layer4,
+    ),
+    UnsupportedFeatures: sets.New(
+        suite.SupportHTTPResponseHeaderModification,
+        suite.SupportHTTPRouteMethodMatching,
+        suite.SupportHTTPRouteQueryParamMatching,
+    ),
+    // other options
+})
+require.NoError(t, err, "misconfigured conformance test suite")
+cSuite.Setup(t)
+
+for i := 0; i < len(tests.ConformanceTests); i++ {
+    test := tests.ConformanceTests[i]
+    test.Run(t, cSuite)
+}
+```
+
+> **NOTE**: `UnsupportedFeatures` must match support levels. Disabling features
+> which are in core will emit a warning and wont be reportable.
+
+> **NOTE**: In the future we may consider expanding the options to configure
+> and run the conformance test suite using a container image and/or via a
+> configuration file, however it was decided to hold off from doing any of
+> those in this iteration to avoid it getting to big (see the
+> [alternatives](#alternatives-considered) section for more thoughts).
+
+Some implementations may support more or less extended features than others,
+so in some cases it could be cumbersome to have to list ALL features that you
+_don't_ support so we optionally and inversely allow `SupportedFeatures` so
+you can pick which option makes sense to you, and under the hood the
+expressions will compile to the same overall list:
+
+```go
+cSuite, err := suite.New(suite.Options{
+    GatewayClassName: "acme",
+    Profiles: sets.New(
+        Layer7,
+        Layer4,
+    ),
+    SupportedFeatures: sets.New(
+        suite.SupportHTTPRouteMethodMatching,
+    ),
+    // other options
+})
+```
+
+> **NOTE**: The `UnsupportedFeatures` and `SupportedFeatures` fields are 
+> mutually exclusive.
+
+Once an implementation has integrated with the conformance test suite, they can
+move on to [certification](#certification) to report the results.
+
+[go-test]:https://go.dev/doc/tutorial/add-a-test
+[go]:https://go.dev
+[lib]:https://pkg.go.dev/sigs.k8s.io/gateway-api@v0.6.1/conformance/utils/suite
+
+### Certification
+
+Implementations will be able to report their conformance testing results using
+our [reporting process](#reporting-process). Implementations will be able to
+visibly demonstrate their conformance results on their downstream projects and
+repositories using our [certification process](#certification-process).
+
+#### Reporting Process
+
+When conformance tests complete a log message will be emitted:
```

Comment:
We may have been thinking this separately and simultaneously, because I just recently added a TODO for this:

75ec95f8e573dc6602020c3b8ecb99781bed75df

Let me know if that seems adequate :+1:

Reply:
yep that sounds great 👍🏽